AirCasting/SessionViews/ThresholdSettings/ThresholdsSettingsViewModel.swift

```diff
@@ -10,32 +10,29 @@ class ThresholdSettingsViewModel: ObservableObject {
     @Published var thresholdMedium = ""
     @Published var thresholdHigh = ""
     @Published var thresholdVeryHigh = ""
-    let initialThresholds: [Int32]
+    let initialThresholds: ThresholdsValue
     
-    init(initialThresholds: [Int32]) {
+    init(initialThresholds: ThresholdsValue) {
         self.initialThresholds = initialThresholds
     }
     
-    private func convertToFloat(value: String) -> Float {
-        let floatValue = Float(value) ?? 0
-        return floatValue
+    func resetToDefault() -> ThresholdsValue {
+        return initialThresholds
     }
     
-    func resetToDefault() -> [Float] {
-        var newInitial: [Float] = []
-        initialThresholds.forEach { value in
-            newInitial.append(Float(value))
-        }
-        return newInitial
-    }
-    
-    func updateToNewThresholds() -> [Float] {
+    func updateToNewThresholds() -> ThresholdsValue {
         let stringThresholdValues = [thresholdVeryHigh, thresholdHigh, thresholdMedium, thresholdLow, thresholdVeryLow]
-        var newThresholdValues: [Float] = []
+        var newThresholdValues: [Int32] = []
         for value in stringThresholdValues {
-            let convertedValue = convertToFloat(value: value)
+            let convertedValue = convertToInt(value)
             newThresholdValues.append(convertedValue)
         }
-        return newThresholdValues.sorted { $0 < $1 }
+        let sortedValue = newThresholdValues.sorted { $0 < $1 }
+        return ThresholdsValue(array: sortedValue)
```

Comment:
We are using an array here to sort the values, yes? Since we are creating the array here, and we are sure it has only 5 values, probably it is better to split it to separate values here and use the other initializer, and remove this initializer from ThresholdValue struct.